TPP.Inputting/Emulator Scripts/Desmume/JSON.lua

```diff
@@ -0,0 +1,893 @@
+-- -*- coding: utf-8 -*-
+--
+-- Simple JSON encoding and decoding in pure Lua.
+--
+-- Copyright 2010-2013 Jeffrey Friedl
+-- http://regex.info/blog/
+--
+-- Latest version: http://regex.info/blog/lua/json
+--
+-- This code is released under a Creative Commons CC-BY "Attribution" License:
+-- http://creativecommons.org/licenses/by/3.0/deed.en_US
+--
+-- It can be used for any purpose so long as the copyright notice and
+-- web-page links above are maintained. Enjoy.
+--
+local VERSION = 20140116.10  -- version history at end of file
+local OBJDEF = { VERSION = VERSION }
```

Comment:
`Desmume/JSON.lua` and `BizHawk/JSON.lua` appear to be the same?

Reply:
Yes, but they're included in both places because I don't expect people to keep the lua folder structure intact.

Reply:
hmm okay, fine by me